README.md

```diff
@@ -3783,6 +3814,7 @@ Subscribes for specific published data items.
 |`fuelRange`|Boolean|False|The estimate range in KM the vehicle can travel based on fuel level and consumption|
 |`externalTemperature`|Boolean|False|The external temperature in degrees celsius|
 |`turnSignal`|Boolean|False|See TurnSignal|
+|`gearStatus`|Boolean|False|See GearStatus|
 |`prndl`|Boolean|False|See PRNDL|
```

Comment:
We should add a note in `prndl` stating that the parameter is deprecated
```suggestion
|`prndl`|Boolean|False|See PRNDL. This parameter is deprecated and it is now covered in `gearStatus`|
```
or something similar. Please also add similar notes for the `prndl` param in the other VehicleData RPCs.